modules/drivers/zlib/wscript

```diff
@@ -103,36 +103,41 @@ def build(bld):
         # get compression to work, not decompression.
         minizipDir = os.path.join('contrib', 'minizip')
         minizipIncludes = ['.', minizipDir]
+        print('Building mijnizip')
         minizip = bld(features='c c%s add_targets' % env['LIB_TYPE'] or 'stlib',
                       includes=minizipIncludes,
                       export_includes=minizipIncludes,
                       target='minizip',
                       source=[os.path.join(minizipDir, 'ioapi.c'), 
                               os.path.join(minizipDir, 'zip.c')],
                       path=driverNode,
+                      lib=os.path.join(os.path.join(os.getcwd(), env['install_libdir'], 'z')),
```

Comment:
Why is this needed (or similarly, how did this work before)?

Reply:
My understanding is that this tells you where to look for shared libraries that you're linking in. Since we weren't trying to build that before, it would not have caused problems.